python_modules/libraries/dagster-airlift/dagster_airlift/test/airflow_test_instance.py

```diff
@@ -281,6 +281,7 @@ def make_task_instance(
     start_date: datetime,
     end_date: datetime,
     logical_date: Optional[datetime] = None,
+    try_number: int = 1,
```

Comment:
what's this for?

Reply:
I think this ends up needed in a different PR not sure why I added it here. But essentially task try number is needed to properly query logs